policy/observers.md

```diff
@@ -0,0 +1,22 @@
+# Observers
+
+**Administrator:** A Working Group in Electron Governance or a company represented in a Working Group wishing to introduce an Observer into their Working Group.
```

Comment:
I think instead of `Administrator`, we can use either `maintainer` or `working group` as defined in https://github.com/electron/governance#definitions

Reply:
Any suggestion on how to reword this with the existing definitions to capture the same stipulations?

Reply:
I take it back. I think it's fine as Administrator after rereading the whole doc it makes sense.